packages/wdio-utils/tests/node/index.test.ts

```diff
@@ -418,4 +424,60 @@ describe('startWebDriver', () => {
             )
         )
     })
+    it('should add a unique user-data-dir on Windows for Chrome workers', async () => {
+        // Change the mocked OS to Windows
+        const os = await import('node:os')
+        // @ts-ignore
+        os.default.__setPlatform('win32')
```

Comment:
No need to dynamically import this here. Vitest is smart enough to resolve the dependency afterall. So you can just do:

```ts
import os from "node:os"
```

at the top and change this to:
```suggestion
        // @ts-ignore
        os.__setPlatform('win32')
```